### ISSUE TITLE
Per Pixel multinomial logistic loss

### ISSUE THREAD

**Issue by DaniSch93** [keras-team/keras#7897]
Hey,
i built a Fully Convolutional Network for Semantic Segmentation in Keras. I used my own ConvNet which I trained from scratch to classify objects (Its working fine and has a quite good accuracy for the data set). A per pixel multinomial logistic loss is suggested for the FCN. For training I do have a picture and a label picture of same size, which is actually just the same picture segmented. I tried quite every loss function and read a lot about it. I know in Caffee is a loss function which would just do what i want. It's called Multinominal Logistic Loss Layer:
(http://caffe.berkeleyvision.org/doxygen/classcaffe_1_1MultinomialLogisticLossLayer.html#details). 
For my case I'd rather need the softmax version of it, which is also in Caffee (http://caffe.berkeleyvision.org/doxygen/classcaffe_1_1SoftmaxWithLossLayer.html).

So my issue is to reproduce such a loss function. Is it even possible?
